### ISSUE TITLE
MUC participant list is always shown at first

### ISSUE THREAD

**Issue by poVoq** [poVoq/conversejs-ubports#14]
Issues related to this upstream:
https://github.com/conversejs/converse.js/issues/1642
https://github.com/conversejs/converse.js/issues/1416

Possible workaround:
```
  converse.initialize({
    ...
  }).then(() => {
    var toggleButton = document.getElementsByClassName("toggle-occupants fa-angle-double-right")[0];
    if (toggleButton) {
      toggleButton.click();
    }
  });
```
but
"In Converse 5.x, the class name is `toggle-occupants float-right fa fa-angle-double-right` but it seems not to work for me. Maybe this code needs to be called after the nickname is entered and the join completes?"

and it seems to not always trigger.

**Comment by poVoq** [comment]
There is now a fix for this upstream. Will be solved with the next version.

**Comment by poVoq** [comment]
Fixed in unreleased 7.0 version.
